### PR TITLE
Add decedent fetus

### PR DESCRIPTION
**Pull request by ptran265**
- added patientDecedentFetus profile and examples (moved from VRCL)
- added multipleBirthTotal, parentReportedAgeAtDelivery, and PatientFetalDeath extensions to patientDecedentFetus 

It was a clean build at one point but I couldn't run it again after adding patientFetalDeath extension since VRCL needs to be updated first (PR in VRCL pending)

### CODE REVIEW

**Review by rpassas — APPROVED** [review]

File: input/fsh/profiles/PatientDecedentFetus.fsh
Comment (saulakravitz) [inline input/fsh/profiles/PatientDecedentFetus.fsh:1]: Perhaps the multiple births should be added to PatientVitalRecords?

File: input/fsh/rulesets/DEF_Rulesets.fsh [hunk 1]
```diff
@@ -125,3 +125,30 @@ RuleSet: CategorySlicingRules
...
+  * ^definition = "If not single birth, specify born 1st, 2nd, etc. – For multiple deliveries, the order this infant was delivered in the set. Include all live births and fetal losses."
+// Add Edit Flags, a la VRDR 
+* multipleBirth[x].extension contains
+    BypassEditFlag named bypassEditFlag 0..1
+* multipleBirth[x].extension[bypassEditFlag]
+  * value[x] from ValueSetPluralityEditFlagsVitalRecords (required)
+  * value[x] only CodeableConcept
+  * value[x]
+    * ^short = "To reflect the relevant edit possibilities for plurality."
+    * ^binding.description = "Plurality Edit Flags (NCHS)"
```
Comment (saulakravitz) [inline input/fsh/rulesets/DEF_Rulesets.fsh]: I don't think this is right...
MultipleBirth should have two expensions:
  1) edit flag
  2) [http://hl7.org/fhir/StructureDefinition/patient-multipleBirthTotal](http://hl7.org/fhir/extensions/1.0.0/StructureDefinition-patient-multipleBirthTotal.html)

File: input/fsh/rulesets/DEF_Rulesets.fsh [hunk 2]
```diff
@@ -125,3 +125,30 @@ RuleSet: CategorySlicingRules
...
+  * ^short = "If not single birth - born first, second, third, etc."
+  * ^definition = "If not single birth, specify born 1st, 2nd, etc. – For multiple deliveries, the order this infant was delivered in the set. Include all live births and fetal losses."
+// Add Edit Flags, a la VRDR 
+* multipleBirth[x].extension contains
+    BypassEditFlag named bypassEditFlag 0..1
+* multipleBirth[x].extension[bypassEditFlag]
+  * value[x] from ValueSetPluralityEditFlagsVitalRecords (required)
+  * value[x] only CodeableConcept
+  * value[x]
+    * ^short = "To reflect the relevant edit possibilities for plurality."
```
Comment (saulakravitz) [inline input/fsh/rulesets/DEF_Rulesets.fsh]: Unless this is used multiple times, it should just be incorporated into PatientVitalRecords, or PatientChildVitalRecords.

**Review by saulakravitz — CHANGES_REQUESTED** [review]
See my comments

**Review by saulakravitz — APPROVED** [review]

**Pull request merged**